docs/reference/integrations/bigquery.rst

```diff
@@ -4,41 +4,65 @@
 BigQuery
 ##############
 
-To add a BigQuery datasource do this:
+How to set up BigQuery as a DataSource
+======================================
+
+To add BigQuery datasource do this:
+
+Follow the `Google Cloud library guide <https://googleapis.dev/python/google-api-core/latest/auth.html>`_
+for authentication.
+
+Install the pybigquery package for the BigQuery sqlalchemy dialect (``pip install pybigquery``)
+
+
+Use the CLI:
 
 1. Run ``great_expectations datasource new``
-2. Choose the *SQL* option from the menu.
-3. When asked which SQLAlchemy driver to use, enter ``other``.
-4. Consult the `PyBigQuery <https://github.com/mxmzdlv/pybigquery`_ docs
+2. Choose "Big Query" from the list of database engines, when prompted.
+3. Consult the `PyBigQuery <https://github.com/mxmzdlv/pybigquery`_ docs
    for help building a connection string for your BigQuery cluster. It will look
    something like this:
 
     .. code-block:: python
 
         "bigquery://project-name"
 
+   If you want GE to connect to one of the Google's public datasets, the URL should be:
```

Comment:
Please add an example for specifying one's own dataset name in URL as well, and whether that affects the *requirement* to include project/schema(dataset?)/table as a triple in queries/table names.

Reply:
Addressed.